apps/src/templates/feedback/LevelFeedbackEntry.jsx

```diff
@@ -13,196 +14,170 @@ const getElementHeight = element => {
   return ReactDOM.findDOMNode(element).offsetHeight;
 };
 
-const visibleCommentHeight = 40;
+const visibleCommentHeight = 46;
+
+function LevelFeedbackEntry({feedback}) {
+  const {
+    id,
+    created_at,
+    comment,
+    performance,
+    review_state,
+    seen_on_feedback_page_at,
+    student_first_visited_at,
+    is_awaiting_teacher_review
+  } = feedback;
+
+  const feedbackSeenByStudent =
+    seen_on_feedback_page_at || student_first_visited_at;
+
+  const commentExists = comment.length > 2;
```

Comment:
Do you know why this is `>2`?  (I see that it is logic carried over from the previous implementation.)

Reply:
I'm not sure. Must have been an issue at some point that there were accidental or empty string comments?